GOVERNANCE.md

```diff
@@ -0,0 +1,146 @@
+
+* Whether pull requests are merged or retained in the event of a dispute.
+* Who should become a committer or member of the steering committee.
+* Whether any changes are necessary to the project's governance or other assets in this repository.
+
+Debezium's steering committee members are listed in the [COMMITTEE.yml](./committee.yml).
+
+## Voting rules and procedures
+
+The Debezium project uses a voting system to guarantee no single member or entity can dominate the project, and ensures continued success and diversity in the project's forward vision.
+
+The formal voting process involves the creation of a GitHub issue or pull request.
+A vote is indicated by specifying `yes` or `no` on the issue or pull request, which can include the thumbs-up or thumbs-down emojis.
+After a defined period of time, votes are tallied, and the outcome is published.
+
+Debezium uses several voting mechanisms and procedures for various purposes.
+
+* [Pull request approvals](#pull-request-approvals)
+* [Lazy consensus votes](#lazy-consensus-votes)
+* [Explicit majority votes](#explicit-majority-votes)
+
+Debezium also relies on the notion of **binding** and **non-binding** votes.
+
+A **binding vote** is one cast by specific individuals with given roles based on the voting procedure, who's vote is used to calculate the outcome.
+A **non-binding vote** is one cast by individuals who do not have binding vote roles, and these votes do not count toward the outcome, but are used to guide those with binding votes to a resolution that best fits all involved.
+
+The following describes for each procedure, those who may vote, how the votes are recorded, and duration of the vote.
+
+### Pull request approvals
+
+Pull requests against most repositories may be merged after receiving at least two positive committer votes.
+Voting for pull requests can be done using a comment or approving the pull request on GitHub.
+If the pull request author is a member of the Debezium steering committee, this counts as one of the two required votes.
+
+Pull requests that are small, low-risk, uncontroversial, or prefixed with `[ci]` or `[docs]` may freely be merged by any committer without any votes.
+Examples of uncontroversial changes may include:
+
+* Refactoring redundant code within a small number (2-5) of files that yields no behavior change.
+* Adding or changing logged output.
+* Adding or improving test suite code with no loss of test coverage.
+
+Whenever in doubt about the pull request's scope and risk, use the two positive vote requirement.
+
+### Lazy consensus votes
+
+Lazy consensus votes use the **+1**, **0**, or **-1** votes where:
+
+* `+1` indicates **yes** or the **affirmative**
+* `-1` indicates **no** or the **negative**
+* `0` indicates you have no specific opinion
+
+The voting process concludes when there are at least three `+1` binding votes, and no `-1` binding votes.
```

Comment:
The "no -1" rule may cause some indefinite blocking of progress (it has burned out folks e.g. in the ASF before). Could we have an option that in case a lazy consensus vote is blocked for a longer period of time, that at the discretion of the steering committee an explicit majority vote over that matter can be hold in order to resolve the matter?

Reply:
Agreed. 

I have also personally submitted proposals to other projects in foundations using similar processes, and while the proposal gets :+1: from non-maintainers, they just sit idle with no activity from maintainers, for months, and some for years. I know I have found that equally frustrating. I can't imagine that would ever be the case for us, but should we add anything in the document around this to give contributors some expectation?